classes/Http/Form/SignupForm.php

```diff
@@ -112,13 +108,6 @@ public function validateSpeakerPhoto(): bool
         return true;
     }
 
-    /**
-     * Method that applies validation rules to email
-     *
-     * @return bool
-     *
-     * @internal param string $email
-     */
     public function validateEmail(): bool
```

Comment:
All of these validation methods currently have `public` visibility, however, they only currently need to be `public` because they are directly invoked by tests. I suggest to reduce the visibility to `private` and modify the tests (in a separate PR).